MapboxVision/VideoStream/VisionManager.swift

```diff
@@ -631,8 +631,11 @@ public final class VisionManager {
         dependencies.core.config.useDetection = operationMode.usesDetection
         
         dependencies.recorder.savesSourceVideo = operationMode.savesSourceVideo
+        dependencies.recorder.videoSettings = operationMode.videoSettings
```

Comment:
Seems like don't want to continue recording with an old settings on operation mode changing, so we may want to immediately restart the recording session here.